rocksdb/_rocksdb.pyx

```diff
@@ -1112,24 +1112,6 @@ cdef class ColumnFamilyOptions(object):
         def __set__(self, value):
             self.copts.max_bytes_for_level_multiplier_additional = value
 
-    property soft_rate_limit:
```

Comment:
Some tests are failing now because they try to access these properties. Can you adjust those too?

Reply:
Yep.